DOCKER/Dockerfile

```diff
@@ -17,7 +17,7 @@ ENV JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
 ENV JRE_HOME=/usr/lib/jvm/java-7-openjdk-amd64
 
 RUN mkdir /deploy
-RUN git clone https://github.com/chrismattmann/imagecat.git
+RUN git clone -b docker-allow-override-of-list-file https://github.com/danlamanna/imagecat.git
```

Comment:
I will revert this particular line when it's ready to merge into master, but for testing purposes this is required until then.